src/components/DeviceDetails/DeviceDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import { useMemo } from "react";
+import { get } from "lodash";
+import { Title, Property } from "@deskpro/app-sdk";
+import { useExternalLinks } from "@/hooks";
+import { getOS, getHumanCapacity } from "@/utils";
+import { Container, LansweeperLogo } from "@/components/common";
+import type { FC } from "react";
+import type { DeviceType } from "@/types";
+
+export type Props = {
+  device: DeviceType;
+};
+
+const DeviceDetails: FC<Props> = ({ device }) => {
+  const { getDeviceLink } = useExternalLinks();
+  const site = device.site;
+  const link = useMemo(() => {
+    return getDeviceLink(site?.name, device.key);
+  }, [site?.name, device.key, getDeviceLink]);
+  const os = getOS(get(device, ["operatingSystem"]));
+  const capacity = getHumanCapacity(get(device, ["diskPartitions"]));
+
+  return (
+    <Container>
+      <Title
+        title={get(device, ["assetBasicInfo", "name"])}
+        {...(!link ? {} : { icon: <LansweeperLogo/> })}
+        {...(!link ? {} : { link })}
+      />
```

Comment:
```
{...(link ? { icon: <LansweeperLogo/>, link, } : {} )}
```
These two ifs can be combined to one. Also I would reorder it for the fewest negatives as it's generally easier to read. I.E. `If there is a link, use it else do nothing` is easier than `if there is not a link, don't show it, else show it.

Reply:
done

Reply:
Already fixed.

Reply:
When combined into one object, it becomes less readable